common/views/components/ImageViewer/ImageViewer.js

```diff
@@ -147,18 +159,23 @@ const ImageViewer = ({ id, contentUrl, infoUrl, width }: ImageViewerProps) => {
 
   return (
     <Fragment>
-      <Image
+      <IIIFResponsiveImage
         width={width}
-        contentUrl={contentUrl}
-        lazyload={false}
-        sizesQueries="(min-width: 860px) 800px, calc(92.59vw + 22px)"
-        alt=""
-        clickHandler={() => {
-          handleViewerDisplay('Image');
-        }}
-        zoomable={viewButtonMounted}
-        defaultSize={800}
-        extraClasses="margin-h-auto width-auto full-height full-max-width block"
+        height={height}
+        src={src}
+        srcSet={srcSet}
+        sizes={`(min-width: 860px) 800px, calc(92.59vw + 22px)`} // FIXME: do this better
+        extraClasses={classNames({
+          'block h-center': true,
+          [spacing({ s: 2 }, { margin: ['bottom'] })]: true,
+          'cursor-zoom-in': viewButtonMounted,
+        })}
+        lang={lang}
+        clickHandler={() => handleViewerDisplay('Image')}
+        alt={
+          (canvasOcr && canvasOcr.replace(/"/g, '')) ||
+          'no text alternative is available for this image'
```

Comment:
Out of interest - is this going to be useful, especially if repeated many times?

Reply:
Good question. We have to output something and we don't have a way to e.g. announce 'no images you're about to look at in this set have alternative text'. But we can shorten it down to `no text alternative` for now. Drafting accessible copy is a good one – I expect we'd need to seek help from the content team.

Reply:
>  from the content team 

Or Orla and her team?

Interesting thought on announcing that there is no text for _any_ image over always announcing it thought, something that could be nice to test.